src/components/donation-form/AutofillCategory.tsx

```diff
@@ -0,0 +1,69 @@
+import { ItemResponse } from '@/types/items';
+import { Autocomplete, TextField } from '@mui/material';
+import { useState } from 'react';
+
+interface Item {
+  ItemOptions: ItemResponse[];
+  onCategorySelect: (category: string) => void;
+  value: string;
+}
+
+export default function AutofillCategory(props: Item) {
+  const [itemOptions] = useState<ItemResponse[]>(props.ItemOptions);
```

Comment:
I would add the same filter that you have for item to category rather than just changing the category strait away